ccl/telemetryccl: TestTelemetry failed

ccl/telemetryccl.TestTelemetry [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2959128&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2959128&tab=artifacts#/) on master @ [cbb3db2e5b6e8327e68182f9396d591804b722ec](https://github.com/cockroachdb/cockroach/commits/cbb3db2e5b6e8327e68182f9396d591804b722ec):


```
=== RUN   TestTelemetry
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestTelemetry759830811
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestTelemetry
    telemetry_test.go:48: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestTelemetry759830811
--- FAIL: TestTelemetry (23.58s)
=== RUN   TestTelemetry/multiregion/server
    telemetry.go:92: 
        testdata/telemetry/multiregion:414: SET override_multi_region_zone_config = true;
        ALTER TABLE t9 SET LOCALITY GLOBAL;
        SET override_multi_region_zone_config = false
        expected:
        sql.multiregion.zone_configuration.override.system.table  1
        
        found:
        sql.multiregion.zone_configuration.override.system.table  3
        --- FAIL: TestTelemetry/multiregion/server (19.57s)
=== RUN   TestTelemetry/multiregion
    --- FAIL: TestTelemetry/multiregion (21.56s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestTelemetry PKG=./pkg/ccl/telemetryccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</p>
</details>
/cc @cockroachdb/obs-infra @otan
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/telemetryccl.TestTelemetry [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2967184&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2967184&tab=artifacts#/) on master @ [7fb0684f54abd920e7e392c87bd4ec049c494ed2](https://github.com/cockroachdb/cockroach/commits/7fb0684f54abd920e7e392c87bd4ec049c494ed2):


```
=== RUN   TestTelemetry
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestTelemetry208762824
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestTelemetry
    telemetry_test.go:48: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestTelemetry208762824
--- FAIL: TestTelemetry (30.57s)
=== RUN   TestTelemetry/multiregion/server
    telemetry.go:92: 
        testdata/telemetry/multiregion:414: SET override_multi_region_zone_config = true;
        ALTER TABLE t9 SET LOCALITY GLOBAL;
        SET override_multi_region_zone_config = false
        expected:
        sql.multiregion.zone_configuration.override.system.table  1
        
        found:
        sql.multiregion.zone_configuration.override.system.table  3
        --- FAIL: TestTelemetry/multiregion/server (26.39s)
=== RUN   TestTelemetry/multiregion
    --- FAIL: TestTelemetry/multiregion (28.23s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestTelemetry PKG=./pkg/ccl/telemetryccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</p>
</details>
/cc @cockroachdb/obs-inf @otan
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTelemetry.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/telemetryccl/testdata/telemetry/multiregion
+++ b/pkg/ccl/telemetryccl/testdata/telemetry/multiregion
@@ -346,10 +346,10 @@ INSERT INTO t7 VALUES (1),(2),(3);
 EXPORT INTO CSV 'nodelocal://0/t7' FROM TABLE t7;
 ----
 
-feature-counters
+feature-usage
 IMPORT INTO t7 CSV DATA ('nodelocal://0/t7/export*.csv')
 ----
-sql.multiregion.import  1
+sql.multiregion.import
 
 # Test for locality optimized search counter.
 feature-allowlist
@@ -375,63 +375,63 @@ feature-allowlist
 sql.multiregion.zone_configuration.override.*
 ----
 
-feature-counters
+feature-usage
 SET override_multi_region_zone_config = true;
 ALTER TABLE t9 CONFIGURE ZONE USING num_replicas=10;
 SET override_multi_region_zone_config = false
 ----
-sql.multiregion.zone_configuration.override.user  1
+sql.multiregion.zone_configuration.override.user
 
 # Note that this case illustrates that once the session variable is set, we'll
 # count all instances where a zone configuration is modified, even if that
 # modification didn't strictly require overriding.
-feature-counters
+feature-usage
 SET override_multi_region_zone_config = true;
 ALTER TABLE t9 CONFIGURE ZONE USING gc.ttlseconds=10;
 SET override_multi_region_zone_config = false
 ----
-sql.multiregion.zone_configuration.override.user  1
+sql.multiregion.zone_configuration.override.user
 
-feature-counters
+feature-usage
 ALTER TABLE t9 CONFIGURE ZONE USING gc.ttlseconds=5
 ----
 
-feature-counters
+feature-usage
 SET override_multi_region_zone_config = true;
 ALTER TABLE t9 SET LOCALITY GLOBAL;
 SET override_multi_region_zone_config = false
 ----
-sql.multiregion.zone_configuration.override.system.table  1
+sql.multiregion.zone_configuration.override.system.table
 
 # Ensure that no counters are set in the case where we're not overriding
-feature-counters
+feature-usage
 ALTER TABLE t9 SET LOCALITY REGIONAL BY TABLE IN PRIMARY REGION;
 ----
 
 # Note that this case illustrates that once the session variable is set, we'll
 # count all instances where a table's zone configuration is modified, even if
 # that modification didn't strictly require overriding.
-feature-counters
+feature-usage
 SET override_multi_region_zone_config = true;
 ALTER TABLE t9 SET LOCALITY GLOBAL;
 SET override_multi_region_zone_config = false
 ----
-sql.multiregion.zone_configuration.override.system.table  1
+sql.multiregion.zone_configuration.override.system.table
 
-feature-counters
+feature-usage
 SET override_multi_region_zone_config = true;
 ALTER DATABASE d CONFIGURE ZONE USING num_replicas=10;
 SET override_multi_region_zone_config = false
 ----
-sql.multiregion.zone_configuration.override.user  1
+sql.multiregion.zone_configuration.override.user
 
-feature-counters
+feature-usage
 ALTER DATABASE d CONFIGURE ZONE USING gc.ttlseconds=5;
 ----
 
-feature-counters
+feature-usage
 SET override_multi_region_zone_config = true;
 ALTER DATABASE d ADD REGION "us-east-1";
 SET override_multi_region_zone_config = false
 ----
-sql.multiregion.zone_configuration.override.system.database  1
+sql.multiregion.zone_configuration.override.system.database

